app/models/wizards/memberships/leave_zusatzsektion.rb

```diff
@@ -34,7 +34,7 @@ def valid?
 
     def save!
       super
-      leave_operation.save!
+      leave_operation.save!.tap { send_confirmation_mail }
```

Comment:
For some reasons `after_save :send_confirmation_mail` does not work. May be an issue in the base wizard?